gateways/adyen/request_builder.go

```diff
@@ -249,3 +252,19 @@ func addIfNonEmpty(value string, key string, data *map[string]string) {
 		(*data)[key] = value
 	}
 }
+
+// extractAdyenStreetFormat extracts adyen street format from generic street address
+//                          returns (streetNumber, streetName) format
+//                          If address does not have leading street number, will return ("", street)
+func extractAdyenStreetFormat(streetAddress string) (string, string) {
+	streetLocation := regexp.MustCompile("^\\d+\\s").FindStringIndex(streetAddress)
```

Comment:
1. If we use one of the find submatch methods ([example](https://pkg.go.dev/regexp#example-Regexp.FindStringSubmatch)) on the regex `^(\d+)\s(.*)`, we can directly extract the street number and street name right?
2. Ah right I forgot about that. Let's use back quotes to make this a raw string literal rather than an interpreted string literal to avoid this confusion and get rid of the double back slashes?